IFC4x3/PropertyConstants/e/EDGEDIRECTION_3$zgmxyLnASwCfGSHfyIwO.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="EDGEDIRECTION_3zgmxyLnASwCfGSHfyIwO" Name="EDGEDIRECTION" UniqueId="fff6ac3b-f15c-4a73-a329-41c469f12e98">
```

Comment:
Where is the edge direction defined?

Reply:
Update to STATIONDIRECTION and changed related descriptions accordingly: https://github.com/bSI-InfraRoom/IFC-Specification/pull/168/commits/791b52ae8b05f2e9592ef4fdee6e010917dd44e1